tests/robot-tests/tests/libs/admin-common.robot

```diff
@@ -310,3 +310,148 @@ user enters text into meta guidance data file content editor
+    user waits until page contains accordion section  ${SUBJECT_NAME}
+    user opens accordion section  ${SUBJECT_NAME}
+    user scrolls to accordion section content  ${SUBJECT_NAME}
+    ${accordion}=  user gets accordion section content element  ${SUBJECT_NAME}
+    ${button}=  user gets button element  Delete files  ${accordion}
+    user clicks element  ${button}
+    user clicks button  Confirm
+
+user approves release for immediate publication
+    user clicks button  Edit release status
+    user waits until h2 is visible  Edit release status
+    user clicks radio   Approved for publication
+    user enters text into element  id:releaseStatusForm-internalReleaseNote  Approved by UI tests
+    user clicks radio   As soon as possible
+    user clicks button   Update status
+    user waits until h2 is visible  Sign off
+    user checks summary list contains  Current status  Approved
+    user waits for release process status to be  Complete    ${release_complete_wait}
+    user reloads page  # EES-1448
+    user checks page does not contain button  Edit release status
+
+user navigates to admin dashboard 
+    user goes to url  %{ADMIN_URL}
+    user waits until h1 is visible   Dashboard
+    user waits until page contains title caption  Welcome Bau1
+
+
+user uploads subject 
+    [Arguments]  ${SUBJECT_NAME}  ${SUBJECT_FILE}  ${META_FILE}
+    user clicks link  Data and files
+    user waits until page contains element  id:dataFileUploadForm-subjectTitle
+    user enters text into element  id:dataFileUploadForm-subjectTitle   ${SUBJECT_NAME}
+    user chooses file   id:dataFileUploadForm-dataFile       ${FILES_DIR}${SUBJECT_FILE}
+    user chooses file   id:dataFileUploadForm-metadataFile   ${FILES_DIR}${META_FILE}
+    user clicks button  Upload data files
+    user waits until h2 is visible  Uploaded data files
+    user waits until page contains accordion section   ${SUBJECT_NAME}
+    user opens accordion section   ${SUBJECT_NAME}
+    ${section}=  user gets accordion section content element  ${SUBJECT_NAME}
+    user checks headed table body row contains  Status           Complete  ${section}  180
+
+user navigates to footnotes
+    user clicks link  Footnotes
+    user waits until h2 is visible  Footnotes
+
+user waits for release to publish 
+    # EES-1007 - Release process status doesn't automatically update
+    user waits until h2 is visible  Sign off
+    user checks summary list contains  Current status  Approved
+    user waits for release process status to be  Complete    ${release_complete_wait}
+    user reloads page  # EES-1448
+    user checks page does not contain button  Edit release status
+
+
+user navigates to Data blocks section 
+    user clicks link    Data blocks
+    user waits until h2 is visible  Data blocks
+    user clicks link  Create data block
+    user waits until h2 is visible  Create data block
+    user waits until h2 is visible   Choose a subject
+
+# probably needs a better name
```

Comment:
I'd remove this comment - if we left this in it would just stay here and nobody would end up renaming the keyword.